ui/src/components/ripple-out/RippleOutSection.js

```diff
@@ -0,0 +1,97 @@
+import { Box, Container, Flex, Text, Center, Button } from '@chakra-ui/react';
+import { useHistory } from 'react-router-dom';
+
+const RippleOutSection = () => {
+  const history = useHistory();
+
+  const handleClick = () => {
+    history.push('/ripple-out');
+  };
+
+  return (
+    <>
+      {/* Hero section - desktop */}
+      <Box
+        w="full"
+        h="100vh"
+        bgImage={`url(${process.env.PUBLIC_URL}/images/ripple-out/ripple-out-hero.png)`}
+        bgSize="cover"
+        bgPos={['right', 'center']}
+        display={['none', 'block']}
+      >
+        <Container maxW="container.xl" h="100%">
+          <Center h="full" flexDir="column">
+            <Flex w="100%" flexDir="column" color="#ffffff">
+              <Text
+                textStyle="darker_grotesque_bold"
+                fontWeight="900"
+                fontSize={['4em', '14em']}
+                lineHeight={['1em', '0em']}
+              >
+                RIPPLE OUT
```

Comment:
Same here, think there's not enough space if this becomes 2 lines on some devices.

<img width="503" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/56828805/fcdb107b-fae5-468b-9cff-504c2d380cc6">

Reply:
i fixed, check again

Reply:
<img width="708" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/56828805/550fd519-51fe-49de-bf5f-b1c12c1027a3">
in ipads it became like this :")